### ISSUE TITLE
Password reset email is blank in 1.4

### ISSUE THREAD

**Issue by amit777** [mautic/mautic#1783]
## Description
When you click "Forgot Password" in login form, the email comes through, however, it looks blank.  After viewing the source of the email, it seems like swift mailer is constructing an email with 2 text/plain parts. One of the parts is blank.  here is an example of the email that is sent:

```
Message-ID: <1d04bbce980bcbeac8b0eae88fdfb3dc@analytics.example.com>
Date: Tue, 24 May 2016 04:19:42 +0000
Subject: Mautic password reset
From: Mautic <mautic@example.com>
To: Admin <admin@example.com>
MIME-Version: 1.0
Content-Type: multipart/alternative;
 boundary="_=_swift_v4_1464063582_8f43f1ca0cb94d0637039846d2710c65_=_"


--_=_swift_v4_1464063582_8f43f1ca0cb94d0637039846d2710c65_=_
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable

Hi Joe,

You requested a password reset. Click the link below to reset=
 your password.

http://analtyics.example.com/index.php/passwordresetconfir=
m?token=3D740961d2583b33b483f53222a23876298d20b1eb57aa0e261a97f06f7afc1abc<=
img style=3D"display: none;" height=3D"1" width=3D"1" src=3D"data:image/gif=
;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=3D" alt=3D"Mautic is open sourc=
e marketing automation" />

--_=_swift_v4_1464063582_8f43f1ca0cb94d0637039846d2710c65_=_
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable



--_=_swift_v4_1464063582_8f43f1ca0cb94d0637039846d2710c65_=_--
```

**Comment by paulop** [comment]
Same problem here =(

**Comment by corradofiore** [comment]
I can confirm the problem.  This is what the message should look like:

```
Message-ID: <a2d7bb5b26d460532715453d3ed5f12d@mautic.linx-marketing.com>
Date: Wed, 15 Jun 2016 14:30:05 +0400
Subject: Mautic password reset
From: FIRSTNAME LASTNAME <user@domain.com>
To: FIRSTNAME LASTNAME <anotheruser@anotherdomain.com>
MIME-Version: 1.0
Content-Type: multipart/alternative;
 boundary="_=_swift_v4_1465986605_b6543b3ee4880979ed5d98f399c4ae4e_=_"


--_=_swift_v4_1465986605_b6543b3ee4880979ed5d98f399c4ae4e_=_
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable

Hi NAME,

You requested a password reset. Click the link below to re=
set your password.

http://HOSTNAME/passwordresetconfi=
rm?token=3Dcese4h4dgd4gdog7gsfi7gsfo8a3shfo8shoss3fss08591b8777be9b7304d036

--_=_swift_v4_1465986605_b6543b3ee4880979ed5d98f399c4ae4e_=_
Content-Type: text/html; charset=utf-8
Content-Transfer-Encoding: quoted-printable

Hi NAME,<br />
<br />
You requested a password reset. Click the link below to re=
set your password.<br />
<br />
http://HOSTNAME/passwordresetconfi=
rm?token=3Dcese4h4dgd4gdog7gsfi7gsfo8a3shfo8shoss3fss08591b8777be9b7304d036

<img style=3D"display: none;" height=3D"1" width=3D"1" src=3D"data:image/gi=
f;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=3D" alt=3D"Mautic is open sour=
ce marketing automation" />

--_=_swift_v4_1465986605_b6543b3ee4880979ed5d98f399c4ae4e_=_--
```

**Comment by escopecz** [comment]
This should be fixed by #2008.
